src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -37,7 +37,13 @@ doLedgerEntry(Context const& context)
         if (!key.parseHex(request.at(JS(index)).as_string().c_str()))
             return Status{Error::rpcINVALID_PARAMS, "malformedIndex"};
     }
-    else if (request.contains(JS(account_root)))
+    if (request.contains(JS(ledger_index)))
+    {
+        if (!request.at(JS(ledger_index)).is_string() ||
```

Comment:
As it is now, if i send true as ledger_index it will crash the application.
If you would fix as_int64() to the intended is_int64() then the condition will always resolve to true as 'ledger_index' can't be both a string and int64 at the same time.